File: app/src/main/java/com/example/myapplication/Utils/MakeLog.java
package com.example.myapplication.Utils;

import android.util.Log;

import com.orhanobut.logger.Logger;

public class MakeLog {

    public static void log(String tag, String msg) {
        Log.e(tag, msg);
    }

    public static void logger(String msg) {
        Logger.e(msg);
    }
}

File: app_kotlin/src/main/java/com/example/myapplication/utils/MakeLog.kt
package com.example.myapplication.utils

import android.util.Log

import com.orhanobut.logger.Logger

object MakeLog {

    fun log(tag: String, msg: String) {
        Log.e(tag, msg)
    }

    fun logger(msg: String) {
        Logger.e(msg)
    }
}
